packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -31,7 +31,8 @@ import { visitXcssPropPath } from './xcss-prop';
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const packageJson = require('../package.json');
 const JSX_SOURCE_ANNOTATION_REGEX = /\*?\s*@jsxImportSource\s+([^\s]+)/;
-const DEFAULT_IMPORT_SOURCE = '@compiled/react';
+const COMPILED_IMPORT_SOURCE = '@compiled/react';
+const DEFAULT_IMPORT_SOURCES = [COMPILED_IMPORT_SOURCE, '@atlaskit/css'];
```

Comment:
I guess it really barely adds much performance or bundle impact—if call it the Atlassian-internal package and it makes sense.  Would just be weird if we changed this over time or didn't actually get `@atlaskit/css` out there.

Reply:
> Would just be weird if we changed this over time or didn't actually get @atlaskit/css out there.

And we can update it if it changes. Also it will get out there ;).

Reply:
Will this impact all consumers, or just Atlassian consumers?

Reply:
Great question! It impacts everyone, by design. So imagine any Ecosystem folks wanting to use the new xcss prop capabilities – they don't have to configure the custom import source to use styles from atlaskit/css (which is the primary API we'll be documenting for usage on ADS site).

Outside of that it's just a no-op, none would be the wiser.